tests/appsec/test_asm_standalone.py

```diff
@@ -104,14 +113,14 @@ def setup_no_appsec_upstream__no_asm_event__is_kept_with_priority_1__from_minus_
     def test_no_appsec_upstream__no_asm_event__is_kept_with_priority_1__from_minus_1(self):
         self.assert_product_is_enabled(self.check_r, self.tested_product)
         spans_checked = 0
-        tested_meta = {"_dd.p.appsec": None, "_dd.p.other": "1"}
+        tested_meta = {self.propagated_tag(): None, "_dd.p.other": "1"}
         tested_metrics = {"_sampling_priority_v1": lambda x: x < 2}
 
         for data, trace, span in interfaces.library.get_spans(request=self.r):
             assert self._assert_tags(trace[0], span, "meta", tested_meta)
             assert self._assert_tags(trace[0], span, "metrics", tested_metrics)
 
-            assert span["metrics"]["_dd.apm.enabled"] == 0  # if key missing -> APPSEC-55222
+            assert span["metrics"][self.propagated_tag()] == 0  # if key missing -> APPSEC-55222
```

Comment:
This is not right, previously was `"_dd.apm.enabled"` and now is `"_dd.p.appsec"`. This value hasn't changed in the spec as far as I know.

Reply:
Good catch. Reverted